src/providers/discord.ts

```diff
@@ -200,29 +312,44 @@ async function snapshotHelpCommandHandler(interaction) {
       'https://github.com/snapshot-labs/brand/blob/master/icon/icon.png?raw=true'
     )
     .addFields(
-      { name: '`/ping`', value: 'Description: Make sure the bot is online.' },
       {
-        name: '`/help`',
-        value: 'Description: List all commands and current notifications.'
+        name: '`/ping` - Make sure the bot is online.',
+        value: '----------------------------------------------------'
       },
       {
-        name: '`/add`',
-        value: `Description: Add notifications on a channel when a proposal start.
+        name: '`/help` - List all commands and current notifications.',
+        value: '----------------------------------------------------'
+      },
+      {
+        name: '`/add` - Add notifications on a channel when a proposal start.',
```

Comment:
This message is misleading, when the guild has subscribed to other events

Reply:
We already have the list of `events`, initialized on line 281, maybe we could reuse that